### ISSUE TITLE
error that was caused by airforce!

### ISSUE THREAD

**Issue by hdsz25** [xtekky/gpt4free#2375]
Hi after updating to the latest version g4f-0.3.4.5, I got below error:
requests.exceptions.SSLError: HTTPSConnectionPool(host='api.airforce', port=443): Max retries exceeded with url: /imagine/models (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))

By the way this issue exists in the previous version as well! After I manually remove the codes related to AirForce then it works!

**Comment by hlohaus** [comment]
I fixed this issue

**Comment by LIZARD-OFFICIAL-77** [comment]
Can we just remove airforce? im trying to use gpt-3.5-turbo and its broken ffs

**Comment by hlohaus** [comment]
The Airforce provider is not mandatory. It functions effectively on my phone. I recommend experimenting with the gpt-4o model. gpt-3.5 is an outdated version.

**Comment by LIZARD-OFFICIAL-77** [comment]
Yes but there are so many cases where gpt-4o is stupid compared to gpt-3.5-turbo. We should find new providers for it.
